aws/templates/segment/segment_bastion.ftl

```diff
@@ -0,0 +1,242 @@
+        }]
+
+        [#assign configSets =
+                getInitConfigDirectories() +
+                getInitConfigBootstrap(component.Role!"") ]
+
+        [#assign fragment =
+            contentIfContent(solution.Fragment, getComponentId(component)) ]
+
+        [#assign contextLinks = getLinkTargets(occurrence, links) ]
+        [#assign _context =
+            {
+                "Id" : fragment,
+                "Name" : fragment,
+                "Instance" : core.Instance.Id,
+                "Version" : core.Version.Id,
+                "DefaultEnvironment" : defaultEnvironment(occurrence, contextLinks),
+                "Environment" : {},
+                "Links" : contextLinks,
+                "DefaultCoreVariables" : true,
+                "DefaultEnvironmentVariables" : true,
+                "DefaultLinkVariables" : true,
+                "Policy" : standardPolicies(occurrence),
+                "ManagedPolicy" : [],
+                "Files" : {},
+                "Directories" : {}
+            }
+        ]
+
+        [#-- Add in fragment specifics including override of defaults --]
+        [#assign fragmentListMode = "model"]
+        [#assign fragmentId = formatFragmentId(_context)]
+        [#assign containerId = fragmentId]
+        [#include fragmentList?ensure_starts_with("/")]
+
+        [#assign environmentVariables = getFinalEnvironment(occurrence, _context).Environment ]
+
+        [#assign configSets +=
+            getInitConfigEnvFacts(environmentVariables, false) +
+            getInitConfigDirsFiles(_context.Files, _context.Directories) ]
+
+        [#if sshEnabled &&
+            (
+                (componentType == "bastion") ||
+                ((componentType == "vpc") && sshInVpc)
+            )]
+
+            [#if deploymentSubsetRequired("iam", true) &&
+                    isPartOfCurrentDeploymentUnit(bastionRoleId) &&
+                    sshStandalone]
+                [@createRole
+                    mode=listMode
+                    id=bastionRoleId
+                    trustedServices=["ec2.amazonaws.com" ]
+                    policies=
+                        [
+                            getPolicyDocument(
+                                ec2IPAddressUpdatePermission() +
+                                    s3ListPermission(codeBucket) +
+                                    s3ReadPermission(codeBucket) + 
+                                    cwLogsProducePermission(computeClusterLogGroupName),
+                                "basic")
+                        ] +
+                        arrayIfContent(
+                            [getPolicyDocument(_context.Policy, "fragment")],
+                            _context.Policy)
+                    managedArns=_context.ManagedPolicy
+                /]
+            [/#if]
+
+            [#if deploymentSubsetRequired("eip", true) &&
+                    isPartOfCurrentDeploymentUnit(bastionEIPId) &&
+                    sshStandalone]
```

Comment:
sshStandalone was used when we were optionally using the nat servers as ssh hosts. Now that we are running bastion in a asg with default of 0, I think it would be cleaner to remove the option of combining the nat server and the ssh server. It is also easier operationally as there is only one ssh ip you have to worry about.

Probably cleaner to remove sshStandalone in a separate PR but if it would make this PR cleaner, please do so here.